docs/_src/api/api/primitives.md

```diff
@@ -463,7 +463,7 @@ def wrong_examples(node: str, n: int = 3, simulated_top_k_reader: int = -1, simu
             "document_id_or_context",
             "answer",
             "document_id_or_answer",
-        ] = "document_id_or_answer", document_metric: str = "recall_single_hit", answer_metric: str = "f1", eval_mode: Literal["integrated", "isolated"] = "integrated", answer_scope: Literal["any", "context", "document_id", "document_id_and_context"] = "any") -> List[Dict]
```

Comment:
Backward compatibility is not important here: let's better have some reasonable default values. 0.5 is a good default threshold for f1-metrics. So let's use this.